tests/test_backend.py

```diff
@@ -24,9 +24,8 @@
 
 from ngraph_onnx.onnx_importer.backend import NgraphBackend
 
-backend_name = pytest.config.getoption('backend', default='CPU')
 # Set backend device name to be used instead of hardcoded by ONNX BackendTest class ones.
-NgraphBackend.backend_name = backend_name
+NgraphBackend.backend_name = pytest.config.getoption('backend', default='CPU')
```

Comment:
It would be more future-proof to make this a list, to allow support for multiple devices at the same time.